build.sbt

```diff
@@ -83,8 +83,6 @@ libraryDependencies ++= Seq(
 
 releaseSettings
 
-shellPrompt := { state => "%s| %s> ".format(GitCommand.prompt.apply(state), version.value)}
-
```

Comment:
I removed this due to the new version of the plugin. 
I couldn't resolve the part of base version but I read that the plugin takes the base version as project.version attribute.